dashboard/test/controllers/api/v1/regional_partners_controller_test.rb

```diff
@@ -278,6 +278,21 @@ class Api::V1::RegionalPartnersControllerTest < ActionController::TestCase
     assert_equal regional_partner.contact_name, JSON.parse(@response.body)['contact_name']
   end
 
+  test 'find regional partner on second geocoder call' do
+    mock_illinois_object = OpenStruct.new(state_code: "IL")
+
+    # Specify the multiple expects calls in reverse order.  We will fail on the
+    # first geocoder lookup, but succeed on the second one.
+    Geocoder.expects(:search).returns([mock_illinois_object])
+    Geocoder.expects(:search).returns([])
```

Comment:
This reverse ordering is a surprise, but a quick test seems to confirm it:
```
  test 'quick test' do
    Geocoder.expects(:blah).returns("1")
    Geocoder.expects(:blah).returns("2")
    Geocoder.expects(:blah).returns("3")

    puts Geocoder.blah
    puts Geocoder.blah
    puts Geocoder.blah
  end
```
Outputs:
```
3
2
1
```
even after upgrading `minitest` from our usual `5.5` to the latest `5.11.3` locally.